src/pudl/output/pudltabl.py

```diff
@@ -537,13 +538,15 @@ def gens_eia860(self, update=False, unit_ids=False):
             pandas.DataFrame: a denormalized table for interactive use.
 
         """
-        if update or self._dfs['gens_eia860'] is None:
+        if update or self._dfs['gens_eia860'] is None \
+                or backfill_tech != self.backfill_eia860_tech:
```

Comment:
what is the impetus for having this flag? is it so that if you change your mind after generating the output it will still regenerate? regardless, can you add some docs for this in the doc string?

Reply:
Sure thing! The point is so that the db will regenerate if you ask it to switch between `backfilled=True` and `backfilled=False` so you aren't stuck with one or the other.

Reply:
okay great! yes that does sound useful.

Reply:
while I'm here, can you explain what the `unit_ids` argument is? I can add a doc string for that too.

Reply:
I believe this adds unit ID's via `assign_unit_ids` so that more or less all of the generators have a PUDL-assigned unit ID

Reply:
It seems like this is conflating attributes of the output object with arguments of the individual output functions, in a way that could give unexpected results. You can end up with a `gens_eia860` dataframe cached in the object which is inconsistent with the attributes of the instance that you created.